BROWSER_README.md

```diff
@@ -36,7 +36,7 @@ We will review your solution in a recent version of Firefox, Chrome, or Safari r
 
 ## Full Description 
 
-Please write a webpage that loads data from `https://swapi.dev/api/planets/`, formats and displays that data in a table.
+Please write a webpage that loads data from `https://swapi.dev/api/planets/`, formats and displays that data in a table. SWAPI documentation can be found [here](https://swapi.dev/documentation)
```

Comment:
Thoughts on this change? Should the candidate be expected to seek out the documentation? Its not hidden, by any means, but providing it might help keep the candidate unblocked on questions related to the data being returned.